### PR TITLE
Update to 2022e

### PR DESCRIPTION
**Pull request by Arishamays1**
# Version bump to `2022e`
- updated to sha256

### CODE REVIEW

**Review by marcoesters — APPROVED** [review]

**Review by varlackc — COMMENTED** [review]
We should try to build `win-64` it seems that the package is skipping the windows builds:

https://github.com/AnacondaRecipes/tzdata-feedstock/blob/4882c8f4ce53290ea3fc864b58963e9cb9a05920/recipe/meta.yaml#L19

**Comment by marcoesters** [comment]
> We should try to build `win-64` it seems that the package is skipping the windows builds:
> 
> https://github.com/AnacondaRecipes/tzdata-feedstock/blob/4882c8f4ce53290ea3fc864b58963e9cb9a05920/recipe/meta.yaml#L19

See here on why not to do this:

https://github.com/AnacondaRecipes/tzdata-feedstock/blob/master/recipe/meta.yaml#L15-L17

https://github.com/AnacondaRecipes/tzdata-feedstock/pull/11

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
   - url: https://data.iana.org/time-zones/releases/tzcode{{ version }}.tar.gz
-    sha256: 3e7ce1f3620cc0481907c7e074d69910793285bffe0ca331ef1a6d1ae3ea90cc
+    sha256: d40280253980e89168e6be4275a852bf9521524d47684de3135b9a5ca387710b
 
 build:
   number: 0
+  noarch: generic
   # This package needs to be noarch because it uses POSIX system APIs.
   # There is no Windows port available and porting to Windows is not trivial.
+  # The Windows build will be skipped when built.
```
Comment (marcoesters) [inline recipe/meta.yaml:18]: I would add "... to avoid Prefect failures."

I would also move that line above "There is no Windows port..."

**Review by marcoesters — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
Will these test commands or part of them work https://github.com/conda-forge/tzdata-feedstock/blob/153db5039b6b70c9f065a3201f2da66c3ca17961/recipe/meta.yaml#L30-L51 ?

**Comment by Arishamays1** [comment]
> 

These tests are unneeded as they do not work and are not compatible with windows As a result, it was removed from the meta.yaml in an earlier commit.

**Comment by marcoesters** [comment]
> Will these test commands or part of them work https://github.com/conda-forge/tzdata-feedstock/blob/153db5039b6b70c9f065a3201f2da66c3ca17961/recipe/meta.yaml#L30-L51 ?

@skupr-anaconda  They are all in `run_tests.sh`

**Pull request merged**